test/unitquat.jl

```diff
@@ -17,6 +17,13 @@ import Rotations: vmat, rmult, lmult, hmat, tmat
     @test QuatRotation(1, 0, 0, 0) isa QuatRotation{Float64}
     @test QuatRotation(1.0f0, 0, 0, 0) isa QuatRotation{Float32}
 
+    # indexing
+    qi = QuatRotation(1, 0, 0, 0)
+    @test qi[1] == 1.0
+    @test qi[6] == 0.0
+    @test qi[5] == 1.0
+    @test_throws BoundsError q1[10]
```

Comment:
```suggestion
    @test_throws BoundsError qi[10]
```
I think it would be less misleading to use the variable name `q3` instead of `qi`.

Reply:
Ah good catch :+1: